blocks/user-licenses/user-actions.js

```diff
@@ -0,0 +1,99 @@
+import execute from '../../scripts/zemax-api.js';
+import { hideModal } from '../../scripts/scripts.js';
+
+function processResponse(event, data, renderViewMethod, hideActionModal) {
+  if (data?.status === 204) {
+    // TODO show success toast message
+    if (hideActionModal) {
+      hideModal(event);
+    }
+    // Re render view after the data is updated
+    if (renderViewMethod) {
+      renderViewMethod(event);
+    }
+  } else {
+    console.log('error ', data);
+  }
+}
+
+export async function addColleague(event, callback) {
+  event.preventDefault();
+  const formNode = event.target.parentNode;
+  const firstName = formNode.querySelector('#first-name').value;
+  const lastName = formNode.querySelector('#last-name').value;
+  const email = formNode.querySelector('#email').value;
+  const jobTitle = formNode.querySelector('#job-title').value;
+  const phone = formNode.querySelector('#phone').value;
+  const personalPhone = formNode.querySelector('#personal-phone').value;
+  const parentcustomerid = localStorage.getItem('parentcustomerid');
+  const data = await execute('dynamics_add_colleague', `&firstname=${firstName}&lastname=${lastName}&emailaddress1=${email}&jobtitle=${jobTitle}&telephone1=${phone}&mobilephone=${personalPhone}&parentcustomerid_account=${parentcustomerid}`, 'POST');
```

Comment:
Nitpick: You could also pass an object here, and then in the `execute` function put it all together with https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

e.g. 
```
const paramsObj = { foo: "bar", baz: "bar" };
const searchParams = new URLSearchParams(paramsObj);
console.log(searchParams.toString()); // "foo=bar&baz=bar"
```